torch/_inductor/compile_fx.py

```diff
@@ -1240,6 +1240,8 @@ def compile_fx_aot(
     inner_compile: _CompileFxCallableEx = compile_fx_inner,
     config_patches: Optional[Dict[str, str]] = None,
 ) -> Union[List[str], str]:
+    assert isinstance(model_, GraphModule), model_
```

Comment:
Nit, msg should be cast to string, right?

Reply:
well assert will repr it so nbd